### PR TITLE
CB-14194 adding a new property to enable multiaz on datalake, now the…

### PR DESCRIPTION
**Pull request by doktoric**
… umsmock enabled datalake multiaz and user can choose whether they would like to deploy dl with multi az or not. The validation checks that it can be enabled only on aws.

See detailed description in the commit message.

### CODE REVIEW

**Review by lnardai — APPROVED** [review]

File: datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- // CB-14194 adding multiaz field to datalake object to request a multiaz aws datalake
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE sdxcluster ADD COLUMN IF NOT EXISTS enablemultiaz boolean;
+UPDATE sdxcluster SET enablemultiaz = FALSE WHERE enablemultiaz is NULL;
+ALTER TABLE sdxcluster ALTER enablemultiaz SET DEFAULT false;
```
Comment (lajosrodek) [inline datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql:6]: Insert `COLUMN` between `ALTER` and `enablemultiaz`.

**Review by lajosrodek — APPROVED** [review]
LGTM. Two minor comments:

File: mock-thunderhead/src/main/resources/application.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ auth:
     event-generation:
       expiration-minutes: 10
     freeipa:
-      multiaz.enable: false
+      multiaz.enable: true
```
[the remark this answers is not in the document]
Comment (doktoric) [reply mock-thunderhead/src/main/resources/application.yml:17]: yes

File: datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql (hunk 1 above)
Comment (doktoric) [reply datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql:6]: ?

Comment (lajosrodek) [reply datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql:6]: Never mind, `COLUMN` after `ALTER` is entirely optional. Fell free to ignore. :smile:

**Pull request merged**